plutus-conformance/test-cases/uplc/evaluation/builtin/semantics/bls12_381_G1_hashToGroup/hash-empty-dst/hash-empty-dst.uplc

```diff
@@ -0,0 +1,10 @@
+-- Check that hashing a random bytestring with an empty DST gives the expected result.
```

Comment:
One thing that worries me here is that with the golden testing infrastructure you can just say `--accept` and override whatever's in the golden file, even if it should never change.  Maybe that's a good reason to include the expected result in the source rather than just having it checked  in, but as mentioned in a previous comment that makes it harder to see what's going on if something goes wrong.